tests/test_remove_mon_from_cluster.py

```diff
@@ -0,0 +1,126 @@
+"""
+A Testcase to remove one or more mons
+when I/O's are happening.
+
+Polarion-ID- OCS-355
+
+"""
+
+import logging
+import re
+import random
+from ocs import ocp, constants
+from resources import pod
+from ocsci.testlib import tier4, ManageTest
+from ocsci.config import ENV_DATA
+from ocs.exceptions import CephHealthException
+from utility.utils import run_cmd
+
+
+log = logging.getLogger(__name__)
+
+DEP = ocp.OCP(
+    kind=constants.DEPLOYMENT, namespace=ENV_DATA['cluster_namespace']
+)
+POD = ocp.OCP(
+    kind=constants.POD, namespace=ENV_DATA['cluster_namespace']
+)
+
+
+def get_mons_from_cluster():
+    """
+    Getting the list of active mons in the cluster
+    if available = 1 then pod is active or else its inactive/
+    does not exist.
+    """
+
+    ret = DEP.get(resource_name='', out_yaml_format=False,
+                  selector='app=rook-ceph-mon')
+    available_mon = re.findall(r'[\w-]+mon-+[\w-]', ret)
+    return available_mon
+
+
+def verify_mons_are_present():
+    """
+    To verify if there are more than 1 mon
```

Comment:
Replace with: "To verify if there are 1 or more.."

Reply:
Done